tests/operator/serverless/verify.go

```diff
@@ -0,0 +1,111 @@
+package serverless
+
+import (
+	"fmt"
+
+	"github.com/kyma-project/serverless/components/operator/api/v1alpha1"
+	"github.com/kyma-project/serverless/tests/operator/serverless/deployment"
+	"github.com/kyma-project/serverless/tests/operator/utils"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func VerifyDeletion(utils *utils.TestUtils) error {
```

Comment:
Is it used anywhere?

Reply:
yup. main.go line 112

Reply:
https://github.com/kyma-project/serverless/pull/898/files#diff-c3d4b45943f382008afe8ea9e6b42f6067c1af6601cab943c4074a15578eb9c6R112